content/200-concepts/100-components/03-prisma-migrate/index.mdx

```diff
@@ -34,122 +34,125 @@ If you are prototyping, consider using the [`db push` command](db-push) - see [S
+      provider = "postgresql"
+      url      = env("DATABASE_URL")
+    }
+    
+    model User {
+      id    Int    @id @default(autoincrement())
+      name  String
+      posts Post[]
+    }
+    
+    model Post {
+      id        Int     @id @default(autoincrement())
+      title     String
+      published Boolean @default(true)
+      authorId  Int
+      author    User    @relation(fields: [authorId], references: [id])
+    }
+    ```
+
+    <Tip>
+
+    You can use [native type mapping attributes](supported-types-and-db-features#mapping-fields-to-a-specific-native-type) in your schema to decide which exact database type to create (for example, `String` can map to `varchar(100)` or `text`).
+
+    </Tip>
+
+    <!-- prettier-ignore-start -->
+
+1.  Create the first migration:
 
    <CodeWithResult showText="Show migration SQL" hideText="Hide migration SQL">
 <cmd>
 
-   ```terminal
-   prisma migrate dev --name init
-   ```
+```terminal
+prisma migrate dev --name init
+```
 
    </cmd>
 <cmdResult>
 
-   ```sql
-     -- CreateTable
-   CREATE TABLE "User" (
-       "id" SERIAL,
-       "name" TEXT NOT NULL,
-
-       PRIMARY KEY ("id")
-   );
-     -- CreateTable
-   CREATE TABLE "Post" (
-       "id" SERIAL,
-       "title" TEXT NOT NULL,
-       "published" BOOLEAN NOT NULL DEFAULT true,
-       "authorId" INTEGER NOT NULL,
-
-       PRIMARY KEY ("id")
-   );
-
-     -- AddForeignKey
-   ALTER TABLE "Post" ADD FOREIGN KEY("authorId")REFERENCES "User"("id") ON DELETE CASCADE ON UPDATE CASCADE;
-   ```
+```sql
+  -- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL,
+    "name" TEXT NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+  -- CreateTable
+CREATE TABLE "Post" (
+    "id" SERIAL,
+    "title" TEXT NOT NULL,
+    "published" BOOLEAN NOT NULL DEFAULT true,
+    "authorId" INTEGER NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+  -- AddForeignKey
+ALTER TABLE "Post" ADD FOREIGN KEY("authorId")REFERENCES "User"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+```
 
    </cmdResult>
 </CodeWithResult>
 
-   Your Prisma schema is now in sync with your database schema and you have initialized a migration history:
+    Your Prisma schema is now in sync with your database schema and you have initialized a migration history:
 
-   ```
-   migrations/
-     └─ 20210313140442_init/
-       └─ migration.sql
-   ```
+    ```
+    migrations/
+      └─ 20210313140442_init/
+        └─ migration.sql
+    ```
 
-1. Evolve your schema by introducing additional fields:
+1.  Evolve your schema by introducing additional fields:
 
-   ```prisma highlight=3;add
-   model User {
-     id       Int    @id @default(autoincrement())
-     jobTitle String
-     name     String
-     posts    Post[]
-   }
-   ```
+    ```prisma highlight=3;add
+    model User {
+      id       Int    @id @default(autoincrement())
+      jobTitle String
+      name     String
+      posts    Post[]
+    }
+    ```
 
-1. Create the second migration:
+1.  Create the second migration:
 
```

Comment:
@carlagn Looks like the issue arises from the `<cmd>` tag here. It doesn't look like it has a corresponding closing one